docs/standard/serialization/system-text-json/source-generation.md

```diff
@@ -17,24 +16,19 @@ ms.topic: how-to
 
 # How to use source generation in System.Text.Json
 
-:::zone pivot="dotnet-7-0,dotnet-6-0"
-
-This article shows how to use the source generation features of [System.Text.Json](overview.md).
-:::zone-end
-
-:::zone pivot="dotnet-5-0,dotnet-core-3-1"
-For information about how to use source generation in System.Text.Json, see [the .NET 6 version of this article](source-generation.md?pivots=dotnet-6-0).
-:::zone-end
-
-:::zone pivot="dotnet-7-0,dotnet-6-0"
+Source generation in System.Text.Json is available in .NET 6 and later versions. Source generation consists of two modes: *metadata collection* and *serialization optimization*.
 
 ## Use source generation defaults
 
 To use source generation with all defaults (both modes, default options):
 
-* Create a partial class that derives from <xref:System.Text.Json.Serialization.JsonSerializerContext>.
-* Specify the type to serialize or deserialize by applying <xref:System.Text.Json.Serialization.JsonSerializableAttribute> to the context class.
-* Call a <xref:System.Text.Json.JsonSerializer> method that takes a <xref:System.Text.Json.Serialization.Metadata.JsonTypeInfo%601> instance. An alternative is to call a <xref:System.Text.Json.JsonSerializer> method that takes a <xref:System.Text.Json.Serialization.JsonSerializerContext> instance.
+1. Create a partial class that derives from <xref:System.Text.Json.Serialization.JsonSerializerContext>.
+1. Specify the type to serialize or deserialize by applying <xref:System.Text.Json.Serialization.JsonSerializableAttribute> to the context class.
+1. Call a <xref:System.Text.Json.JsonSerializer> method that either:
```

Comment:
"either" implies two options, but there are 3

Reply:
According to this, "either" is fine to introduce more than two phrases.

https://www.grammarphobia.com/blog/2017/08/either-neither.html